src/pages/Login/index.js

```diff
@@ -13,6 +13,7 @@ import useTranslation from 'hooks/useTranslation';
 import routesPaths from 'routes/routesPaths';
 import Input from 'components/form/Input';
 import Button from 'components/common/Button';
+import smiles from '../../assets/smiles.svg';
```

Comment:
I'm curious if you can do 
```suggestion
import smiles from 'assets/smiles.svg';
```

Reply:
Oh yes!! It worked. Thank you.